### PR TITLE
closes #392

### LINKED ISSUE #392: Bundler message covers start Android emulator options
**Describe the bug**
Bundler message covers start Android emulator options

**To Reproduce**
1. Have just one emulator with different name than the default one in settings 
1. Run command 'rnv run -p android -r'
2. See error
3. If you hit enter, emulator will starts

**Expected behavior**
Emulator name should be visible

**Screenshots or copy&paste**
Screenshot 2020-01-16 at 08 09 49

**Desktop (please complete the following information):**
 - OS: osx
 - Node Version v12.12.0
 - RNV Version 0.28.17-alpha.2 

**Additional context**
Not happening if there is just one emulator with default name
Not happening if there is more than one emulator

### PR DESCRIPTION
**Pull request by mihaiblaga89**

### CODE REVIEW

File: packages/rnv/src/platformTools/bundler.js [hunk 1]
```diff
@@ -5,43 +5,37 @@ import Config from '../config';
 
 export const isBundlerRunning = async (c) => {
     try {
-        const { data } = await axios.get(`http://127.0.0.1:${c.runtime.port}`);
-        if (data.includes('React Native')) return true;
+        const { data } = await axios.get(`http://127.0.0.1:${c.runtime.port}/index.ios.bundle`);
```
Comment (hosek) [inline packages/rnv/src/platformTools/bundler.js]: is this intentional? Showing loading of ios bundle while the original command was for android can confuse users

Comment (mihaiblaga89) [reply packages/rnv/src/platformTools/bundler.js]: true that, didn't think of it. Will change to reflect platform

**Comment by mihaiblaga89** [comment]
@hosek can you give it a test now? changed it from loading the `.bundle` to loading `.js` since it's faster

File: packages/rnv/src/platformTools/bundler.js (hunk 1 above)
Comment (pavjacko) [reply packages/rnv/src/platformTools/bundler.js]: @mihaiblaga89 android is fixed. ios however will get this:

```
runtime com.apple.CoreSimulator.SimRuntime.iOS-13-3
⚠️  $ rnv run -p ios -t - WARNING: Your specified simulator target true doesn't exists
? Select the simulator you want to launch (Use arrow keys)
❯ iPhone 8 | Phone 📱 | v: iOS-13-3 | udid: E475674B-CCCE-4E4E-B34B-D00FEAE6CC5E
  iPhone 8 Plus | Phone 📱 | v: iOS-13-3 | udid: 676C7C32-B16E-41B6-BF0E-C71429EE9639
  iPhone 11 | Phone 📱 | v: iOS-13-3 | udid: C80138FC-7E80-413F-8511-6081E1405AFB
  iPhone 11 Pro | Phone 📱 | v: iOS-13-3 | udid: 11225487-D2C8-47E2-AE5C-74D9C1C33FB5
  iPhone 11 Pro Max | Phone 📱 | v: iOS-13-3 | udid: E745AE34-941A-4B07-B621-835EDF2D8022
  iPad Pro (9.7-inch) | Tablet 💊 | v: iOS-13-3 | udid: 5DE8C58A-A207-4820-8B5E-FC747BF89712
  iPad (7th generation) | Tablet 💊 | v: iOS-13-3 | udid: EDBFC801-D83B-41DD-A3E4-BA5CE980316C
Loading dependency graph, done.
```

if you run `rnv run -p ios -t`

however I'll merge this fix. there is another branch which will tackles ios simulator related issues

**Pull request merged**